wirehair/CMakeLists.txt

```diff
@@ -10,6 +11,7 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 option(MARCH_NATIVE "Use -march=native option" ON)
+option(BUILD_TESTS "build testsuite" OFF)
```

Comment:
I changed this line in the merge, since I don't want the testsuite. I should probably stop mucking with the CMakeLists and put it in the setup.py...